src/tests/unit/components/order-history-item/order-history-item.spec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { useSelector } from 'react-redux';
+import OrderHistoryItem from '../../../../containers/orders/order-history/order-history-item/order-history-item.js';
+
+jest.mock('react-redux');
+jest.mock(
+  '../../../../containers/orders/order-history/order-history-item/order-history-item.styles.js',
+  () => ({
+    useStyles: () => ({})
+  })
+);
+jest.mock('../../../../utils/date', () => ({
+  getFormatDate: () => {}
+}));
+const order = {
+  _id: '61682716c834282f74bf505d',
+  dateOfCreation: '1634215702441',
+  status: 'CREATED',
+  orderNumber: '1634215702438',
+  items: [
+    {
+      quantity: 1,
+      fixedPrice: [
+        { currency: 'UAH', value: 1950 },
+        { currency: 'USD', value: 74 }
+      ],
+      options: { size: { name: 'S' } },
+      product: {
+        bottomMaterial: {
+          material: {
+            name: [
+              {
+                lang: 'ua',
+                value: 'Шкірзамінник'
+              },
+              {
+                lang: 'en',
+                value: 'Leatherette'
+              }
+            ]
+          }
+        },
+        name: [
+          { lang: 'ua', value: 'Роллтоп синій' },
+          { lang: 'en', value: 'Rolltop blue' }
+        ],
+        model: { sizes: [{ name: 'L' }, { name: 'M' }, { name: 'S' }] },
+        images: { primary: { thumbnail: 'thumbnail_4051pm10kty4dhv5_37.png' } }
+      }
+    }
+  ],
```

Comment:
Please move this to separate ...variables.js file

Reply:
done